tests/UI/specs/UIIntegration_spec.js

```diff
@@ -61,7 +61,7 @@ describe("UIIntegrationTest", function () { // TODO: Rename to Piwik?
         this.title = parentSuite.title; // to make sure the screenshot prefix is the same
 
         it("should load dashboard1 correctly", async function () {
```

Comment:
```suggestion
        it("should load dashboard2 correctly", async function () {
```

For all "should load ..." tests and probably also rename the screenshot files so they match?

Reply:
Good spot. That would definitely trip me as recently I've been relying on the naming convention for the UI test screenshots when looking at the tests.